### ISSUE TITLE
Idea: Different Company Types for different Countries

### ISSUE THREAD

**Issue by Practical-DQ** [benkeen/generatedata#156]
Hi, I have been using the tool on and off for about a month and have made some changes that have made my test data a litte better.
Making the company types specific to the country is a nice touch
I have the records in a sql table and replace the american company type with country specifics, In this case for: Germany, Italy, France and the UK.
The table content is below:

Country CompanyType CompanyTypeLong Description
FRANCE  Affaire personelle
FRANCE  Auto-Entrepreneur       sole proprietorship
FRANCE  EI  Entreprise individuelle entreprise en nom personnel sole proprietorship
FRANCE  EURL    Entreprise Unipersonelle à Responsabilité Limitée    sole proprietorship with limited liability
FRANCE  FCP Fonds commun de placement   unincorporated investment fund
FRANCE  GIE Groupement d’Intéret Économique
FRANCE  SA  Société Anonyme
FRANCE  Sarl    Société à responsibilité limitée   (private company)
FRANCE  SAS Société par actions simplifiée
FRANCE  SASU    Société par actions simplifiée unipersonnelle
FRANCE  SCA Société en commandite par actions
FRANCE  SC  Société civile    partnership with full liability
FRANCE  SCOP    Société coopérative
FRANCE  SCS Société en Commandité Simple
FRANCE  SEM Société d'économie mixte semi-public company
FRANCE  SICAF   Société d'investissement à capital fixe  closed-end investment fund
FRANCE  SICAV   Société d'investissement à capital variable  unit trust /mutual fund.
FRANCE  SNC Société en nom collectif  general partnership
FRANCE  SP  Société en participation
GERMANY AG  Aktiengesellschaft
GERMANY AG & Co. KG
GERMANY AG & Co. KGaA
GERMANY BGB Gesellschaft
GERMANY EG  Eingetragene Genossenschaft
GERMANY EV  Eingetragener Verein    non profit society/association
GERMANY Einzelfirma
GERMANY Einzelunternehmen
GERMANY GbR Gesellschaft burgerlichen Rechts    partnership without a legal name – mainly non-commercial
GERMANY Genossenschaft
GERMANY GmbH    Gesellschaft mbH    Gesellschaft mit beschraenker Haftung
GERMANY GmbH & Co.
GERMANY GmbH & Co. KG
GERMANY GmbH & Co. KGaA
GERMANY GmbH & Co. OHG
GERMANY KG  Kommanditgesellschaft   partnership under a legal name
GERMANY KGaA    Kommanditgesellschaft auf Aktien    limited partnership
GERMANY OHG Offene Handelsgesellschaft  partnership under a legal name
GERMANY Partenreederei
GERMANY PartG   Partnerschaftsgesellschaft
GERMANY Stiftung  
ITALY   Associazione
ITALY   Azienda
ITALY   Consorzio
ITALY   Ente Morale
ITALY   ICDP
ITALY   Municipale
ITALY   Municipale Assicurazioni
ITALY   Società
ITALY   SA  Società Anonima
ITALY   SapA    Società in accomandita per Azioni  limited partnership with shares
ITALY   Sas Società in accomandita semplice    limited partnership
ITALY   Scrl    Società cooperativa a responsabilità limitata co-operative limited liability
ITALY   company
ITALY   SGRpA
ITALY   Società Consortile
ITALY   SCpA    (Società Consortile per Azioni) – joint stock company
ITALY   Società Cooperativa
ITALY   Società di Fatto
ITALY   Società Estera
ITALY   SGR Società di Gestione del Risparmio 
ITALY   Società Individuale
ITALY   SnC Società in nome Collettivo – general partnership
ITALY   SpA Società per Azioni – limited share company
ITALY   SrL Società a responsabilità Limitata limited liability company
ITALY   S.s.    Società semplice
UNITED KINGDOM  CIC Community Interest Company
UNITED KINGDOM  PLC Community Interest Public Limited Company   Community Interest
UNITED KINGDOM  Company Limited by Guarantee
UNITED KINGDOM  Cbc Cwmni Buddiant Cymunedol
UNITED KINGDOM  Ccc Cwmni Buddiant Cymunedol Cyhoeddus Cyfyngedig Cwmni Buddiant
UNITED KINGDOM  Cyf Cyfyngedig  Limited company
UNITED KINGDOM  Foreign Registered Company
UNITED KINGDOM  Friendly Society
UNITED KINGDOM  General Partnership
UNITED KINGDOM  LP  Limited Partnership
UNITED KINGDOM  LLP Limited Liability Partnership
UNITED KINGDOM  Partnership
UNITED KINGDOM  Unltd   Private Unlimited Company
UNITED KINGDOM  Public Corporation
UNITED KINGDOM  Sole Proprietorship
UNITED KINGDOM  Ltd Private Company Limited
UNITED STATES   Co. Company
UNITED STATES   Co-op.  Co-operative
UNITED STATES   Corp    Corporation
UNITED STATES   Inc.    Incorporated
UNITED STATES   Ltd Limited
UNITED STATES   LC  Limited Company
UNITED STATES   Ltd Co. Limited Company
UNITED STATES   LLC Limited Liability Company
UNITED STATES   LLLP    Limited Liability Limited Partnership
UNITED STATES   LLP Limited Liability Partnership
UNITED STATES   LP  Limited Partnership
UNITED STATES   NA  National Association
UNITED STATES   NL  No Liability
UNITED STATES   Partnership
UNITED STATES   PC  Professional corporation
UNITED STATES   PLLC    Professional Limited Liability Company
UNITED STATES   PLLP    Professional Limited Liability Partnership
UNITED STATES   Sole Proprietorship
UNITED STATES   Unlimited

**Comment by MvErven** [comment]
This one is also my request. For the Netherlands. I have been tempering with  some of the files to add them. Also with all the names (male names and female names)

In addition to this question/wish... 
Would it be possible to have a field where we can type (fill in) our own country names, people's names, City names, Provices names and so on? This will then be saved into file. This file can be edited for finetuning the random fileds.

**Comment by benkeen** [comment]
Good call. Or even an interface to manage the data & come up with your own sets.
